### ISSUE TITLE
Changing default voice

### ISSUE THREAD

**Issue by Stanestane** [OpenJabNab/OpenJabNab#48]
Hi, I hope you could help me with the following problem. I world like to charge the voice that TTS plugin uses and I have a problem finding the information about it.

Is it so that Clair is hardcoded? Also do API calls to TTS support voice as parameter? In general I have a bit of a problem of finding information how to configure the plugins.

Thank you in advance!

And thanks for doing such an awesome work.

**Comment by RedoXyde** [comment]
Hi,

The voice depends on the TTS service you are using. When using Acapela, you have multiple choices ( see server/tts/acapela/tts_acapela.cpp ). When using GoogleTranslate, you only have... one.

It is indeed hardcoded, to "julie" or "claire", in different places especially in the plugins (weather, ratp, jokes, msgall, ephemeride, airquality, tv, cinema, clock). It should however be possible to make this setting user-configurable (I did something like that on my fork back in 2011: RedoXyde/OpenjabNab@78544c97dd239d508a2d68829b042cac30b6eea0 ). I however don't know wether or not the Acapela services still work.

The TTS plugin (which provides the say(your_text)  API call) doesn't support the voice as parameter.

The TTS stuff can be updated to be configurable (either by the Admin or the Users), but it requires some work, and I don't think any of the devs have enough spare time to work on that theses days. Feel free to patch it and submit a pull request if you want =) 

Hope it helps,
Best regards

**Comment by Stanestane** [comment]
Hi,
Thanks for the reply. I figured as much myself but I wanted to double check in case I was missing something obvious. Anyhow, I solved my problem by changing the voice in code in plugins which I use and recompiling the server. Acapela seems to work fine at the moment.

Unfortunately, I don' think I would have the time to do a proper contribution by making voice as param. I know that you guys are also probably busy. Great work in even supporting this thing for such a long time.

Thanks again!
